widget/ui/src/components/SwapDetailsPlaceholder/SwapDetailsPlaceholder.tsx

```diff
@@ -37,7 +37,7 @@ export function SwapDetailsPlaceholder(props: PropTypes) {
           <Alert
             type="error"
             title={i18n.t(
-              'swapNotFound',
+              'swapNotFound2',
```

Comment:
Why this change is needed?

Reply:
this had problem with duplicate key and different value,
I fixed it.